app/containers/Products/ProductCreatorContainer.tsx

```diff
@@ -26,11 +27,10 @@ export const ProductCreator: React.FunctionComponent<Props> = ({
     const variables = {
       input: {
         product: {
-          name: e.formData.name,
-          description: e.formData.description,
+          productName: e.formData.name,
+          productDescription: e.formData.description,
           units: e.formData.units,
-          state: 'active',
-          parent: [null],
+          productState: 'DRAFT' as CiipProductState,
```

Comment:
is this cast needed? maybe use `CiipProductState.DRAFT` instead (I forgot how enums work)

Reply:
`CiipProductState` is being exported from createProductMutation.graphql.ts like this: `export type CiipProductState = "ARCHIVED" | "DRAFT" | "PUBLISHED";` Is accessing the enum properties still possible in this way if it's being exported as a TYPE instead of an ENUM? 
It looks here like it can only be used as a type context: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html

Reply:
it looks like we've been casting other enums like CiipApplicationRevisionStatus. (app/components/SubmitApplication.tsx)